blog.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="stylesheet" type="text/css" href="styles.css">  
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.1/css/all.css" integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp" crossorigin="anonymous">
+  <title>Blog</title>
+</head>
+
+<body>
+
+<header>
+<div class="header-holder">
+<section class="left-side-header">
+<h3 class="hidden-header">Left Side Header</h3>
+<img src="images/adoptadog-logo.svg" alt="Adopt a Dog Logo">
+<p>Seattle's premiere dog adoption agency</p>
+</section>
+<section class="right-side-header">
+<h3 class="hidden-header">Right Side Header</h3>
+  <p><i class="fas fa-shopping-cart"></i> $1234.56  </p>
+  <button> Checkout</button>
+    
+    </section>
+</div>    
+</header>
+<nav>
+<ul class="header-nav-list-holder">
+    <li class="header-nav-list-item"><a href="index.html">Home</a></li>
+    <li class="header-nav-list-item">/</li>
+    <li class="header-nav-list-item"><a href="dogs.html">Dogs</a></li>
+    <li class="header-nav-list-item">/</li>
+    <li class="header-nav-list-item"><a href="blog.html">Blogs</a></li>
+    <li class="header-nav-list-item">/</li>
+    </ul>
+    
+    </nav>    
+<!--<div id="wrapper">-->
+<h2 class="page-header">Adoptadog Blog</h2>
+
+    <div class="grid-item-container"> 
+    
+<section class="blog-main-holder">
+<h3 class="hidden-header">Main Blog Holder 1</h3>
```

Comment:
if you use a different HTML5 element for these blog _articles_ you won't need the hidden headers